src/style.css

```diff
@@ -0,0 +1,102 @@
+
+* {
+  box-sizing: border-box;
+}
+
+.search-form {
+  position: relative;
+  margin-top: 20px;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input-big {
+  height: 70px;
+  padding-left: 62px;
+  padding-top: 25px;
+  padding-bottom: 23px;
+}
+
+.input-small {
+  height: 42px;
+  padding-left: 33px;
+  padding-top: 12px;
+  padding-bottom: 10px;
+}
+
+.input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: "Avenir", sans-serif;
```

Comment:
Add font for the whole page

Reply:
inherit font there instead of duplicating styles